lib/classes/class-bootstrap.php

```diff
@@ -667,7 +669,16 @@ public function flush_transients() {
        * Plugin Activation
        *
        */
-      public function activate() {}
+      public function activate() {
+        add_action( 'activated_plugin', array($this, 'redirect_to_splash') );
+        wp_redirect(admin_url('upload.php?page=stateless-setup-wizard&step=splash-screen'));
+      }
+
+      public function redirect_to_splash($plugin =''){
+        if( $plugin == plugin_basename( $this->boot_file ) ) {
```

Comment:
@alimuzzaman we should really have a filter in here we can override later, in certain situations. Perhaps

```php
apply_filter( 'wp_stateless:redirection_url',  plugin_basename( $this->boot_file ), $this );
```
or something similar.  As you add filters/actions, please be sure to document them either in a Wiki file or inline adhering to WordPress standards. See this https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/.